x11-misc/bbcd/bbcd-0.3.1-r2.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Basic CD Player for blackbox wm"
+HOMEPAGE="http://tranber1.free.fr/bbcd.html"
+SRC_URI="http://tranber1.free.fr/${P}.tar.gz"
+LICENSE="GPL-2"
+
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86 ~x86-fbsd"
+IUSE=""
+
+RDEPEND="media-libs/libcdaudio
+	x11-libs/libX11"
```

Comment:
doesn't it like against these? which means it needs to be available at compile time, i.e.
`DEPEND="${RDEPEND}"`